README.md

```diff
@@ -71,6 +71,8 @@ The recommended installation methods for Windows, Linux or macOS are:
 **Note:** we assume you already installed the [PureScript compiler][purescript].
 If not, get it with `npm install -g purescript`, or the recommended method for your OS.
 
+**Note:** If you get networking errors in ubuntu you may need to install the `netbase` package with `apt install netbase`.
+
```

Comment:
We already have [a FAQ question](https://github.com/spacchetti/spago#im-getting-weird-errors-about-libtinfoso5) about other Ubuntu installation problems, so we should either move this to a FAQ question or move that suggestion up here (either is fine by me)

Reply:
Good call. I will fix it.